java/ci/Dockerfile.centos7

```diff
@@ -32,8 +32,6 @@ RUN yum install -y git zlib-devel maven tar wget patch
 RUN mkdir /usr/local/rapids && mkdir /rapids && chmod 777 /usr/local/rapids && chmod 777 /rapids
 
 RUN cd /rapids/ && wget https://dl.bintray.com/boostorg/release/1.72.0/source/boost_1_72_0.tar.gz && \
```

Comment:
This entire RUN statement should be removed.  It's only for downloading and building boost from source (to get PIC object files even in a static lib).